src/components/templates/Auditorium/components/AllSectionPreviews/AllSectionPreviews.scss

```diff
@@ -110,7 +110,10 @@ $all-section-previews-bottom-spacing: 50px;
   }
 
   &__iframe {
-    height: 100%;
     width: 100%;
+    height: 100%;
+    border: thin $black;
+    border-radius: $border-radius--lg;
+    box-shadow: $iframe-box-shadow;
   }
```

Comment:
Hrmm.. still not sure this is right? The only file I expect to see any changes from on this PR should be in `src/scss/customOverrides.scss`?

Reply:
@0xdevalias, the changes are not unrelated. In a past pr, someone (i think @whoaitsaimz) deleted the styles from auditorium v2 so that it doesn't have the glow. I re-added the glow styles to the components (needed to share the iframe-box-shadow through constants.scss) and "re-removed" it through customOverrides.scss.

Reply:
Are you sure they just weren't removed from the code in general? (eg. by @mike-lvov)

Searching for `border-radius: $border-radius--lg;` I don't see it in this file on `staging` either.

Reply:
https://github.com/sparkletown/sparkle/blob/staging/src/components/templates/Auditorium/components/AllSectionPreviews/AllSectionPreviews.scss#L108-L115

And according to the history:

- https://github.com/sparkletown/sparkle/commits/staging/src/components/templates/Auditorium/components/AllSectionPreviews/AllSectionPreviews.scss

This file hasn't been touched in `staging` since it was merged by @mike-lvov in

- https://github.com/sparkletown/sparkle/pull/1251

Reply:
You're right. I'll make sure this only touches v1 stuff and `customOverrides.scss`

Reply:
I basically just reverted the last commit

Reply:
fixed in c78545c

Reply:
Correct, I did not change ANYTHING in any file other than customOverrides.scss ;-)